test/abagnale.js

```diff
@@ -4,6 +4,38 @@ import fs from 'fs';
 import glob from 'glob';
 import path from 'path';
 
+describe('Bad refract input data', () => {
+  it('should not crash', () => {
```

Comment:
I say that because just from looking at the test, I do not know what is bad about this input.